### ISSUE TITLE
Wrong number of traces in new dataset for multi-key

### ISSUE THREAD

**Issue by SpikingNeuron** [ANSSI-FR/ASCAD#12]
Hi All,

I tried the small dataset (281mb) but the number of traces does not match with what is documented.
Please check script the code below. There are 200000 profile traces and 1000 attack traces.
I was expecting 100000 profile traces and 50000 attack traces.

It will be overkill for me to download 71Gb data file and generate the dataset ;)

Below is the code for reference, any help is much appreciated.

```python
import h5py
import numpy as np

# -----------------------------------------------------------
# read single and multi key groups from h5_file
h5_file = h5py.File("ASCAD.h5", "r")
h5_file_sk = h5_file["Attack_traces"]
h5_file_mk = h5_file["Profiling_traces"]

# -----------------------------------------------------------
# read all fields for single key
sk_traces = h5_file_sk["traces"][()]
sk_labels = h5_file_sk["labels"][()].astype(dtype=np.uint8)
sk_key = h5_file_sk["metadata"]["key"]
sk_plaintext = h5_file_sk["metadata"]["plaintext"]
sk_masks = h5_file_sk["metadata"]["masks"]

# -----------------------------------------------------------
# read all fields for multi key
mk_traces = h5_file_mk["traces"][()]
mk_labels = h5_file_mk["labels"][()].astype(dtype=np.uint8)
mk_key = h5_file_mk["metadata"]["key"]
mk_plaintext = h5_file_mk["metadata"]["plaintext"]
mk_masks = h5_file_mk["metadata"]["masks"]

# -----------------------------------------------------------
print(sk_traces.shape)  # outputs: (1000, 1400)
print(mk_traces.shape)  # outputs: (200000, 1400)
```
I was expecting 
```python
print(sk_traces.shape)  # outputs: (50000, 1400)
print(mk_traces.shape)  # outputs: (100000, 1400)
```

**Comment by rb-anssi** [comment]
Hi,

Thanks for reporting the issue: we confirm that there has been a glitch when uploading the  new databases! :-)
We will fix this asap!

Regards,

**Comment by rb-anssi** [comment]
The issue should be fixed with commit  d80787e2dd0e6d67fdfdc0800e48f83fc3fde952 !
